go.mod

```diff
@@ -166,7 +167,7 @@ require (
 	github.com/golangci/revgrep v0.5.3 // indirect
 	github.com/golangci/unconvert v0.0.0-20240309020433-c5143eacb3ed // indirect
 	github.com/google/btree v1.1.3 // indirect
-	github.com/google/flatbuffers v1.12.1 // indirect
+	github.com/google/flatbuffers v23.5.26+incompatible // indirect
```

Comment:
Hmm is there a reason this package was changed?

Reply:
Not entirely sure why it got upgraded, but most likely due to the `go work sync` and the e2e folder, since it also imports v23. Anyway, I’ve downgraded the `swap` module to `v1.21.1` !